linchpin/shell/__init__.py

```diff
@@ -340,19 +340,23 @@ def fetch(ctx, fetch_type, remote, root):
               metavar='TX_ID', required=False,
               help="Display a specific transaction by ID (tx_id)."
                    " Only works with '--view=tx'")
-@click.option('-of', '--output-format', type=str, default="cfg",
+@click.option('--output-format', type=str, default="cfg",
               metavar='output_format', required=False,
               help="Inventory output format")
-@click.option('-outputfile', '--output-file', type=str,
+@click.option('--output-file', type=str,
               metavar='output_file', required=False,
```

Comment:
I'm still wondering why this option can't be a general option as I pointed out previously?

Reply:
@herlo : Output-file is a optional parameter by default the inventory_path from the rundb is considered if not mentioned . It is used when users would like to write inventory file to external path.

Reply:
@samvarankashyap I agree and understand the purpose of the `--output-file` option. I posted this previously [here](https://github.com/CentOS-PaaS-SIG/linchpin/pull/601#discussion_r195868811).

What I am suggesting is that `--output-pinfile` and `--output-file` serve the same purpose for different verbs (subcommands). In this case, `linchpin up` and `linchpin journal`.

Using `--output-file` in one place reduces duplication. See this example:

```
linchpin -vp PinFile.libvirt-mi.template --template-data @Data.libvirt-mi.yml \
--output-pinfile /tmp/PinFile.libvirt-mi.generated up
```
Adding the `--output-file` option to the above command would be an equivalent of `--output-pinfile`:

```
linchpin -vp PinFile.libvirt-mi.template --template-data @Data.libvirt-mi.yml \
--output-file /tmp/PinFile.libvirt-mi.generated up
```
Equally, this could be applied to linchpin journal in this way:

```
linchpin --output-file /tmp/tester.txt journal -t 143 --output-format json \
--output-type inventory --target oso_target
```
The location of `--output-file` is important here. It's a general option so it would come before the `journal` verb. We could do this for `--output-format` as well, since a PinFile could be json or yaml. 

Does this make more sense?